Help: Fix formatting of olark chat notifications.

omarjackman opened the pull request:
This pull request fixes the issue outlined in #1817. It prevents wrapping of subsequent messages after a notification is received.

How to test:

1. Start chatting from http://calypso.localhost:3000/help/contact/
2. Have the operator end the chat (but remain available).
3. Observe that the embedded chat form sends a "your chat has ended" message, but since opers are available, you can type another message to resume the chat.
4. Type a message.
5. Notice how the message no longer sticks to the right of the notification.


#### Before:
![screen shot 2016-01-29 at 7 49 54 pm](https://cloud.githubusercontent.com/assets/1854440/12692613/7f70680a-c6c7-11e5-9fb5-c4f3c99777b0.png)

![screen shot 2016-01-29 at 8 00 18 pm](https://cloud.githubusercontent.com/assets/1854440/12692611/742c4a72-c6c7-11e5-9c3a-5ad8159ae35a.png)

#### After:
![screen shot 2016-01-29 at 7 53 23 pm](https://cloud.githubusercontent.com/assets/1854440/12692665/6652fd6e-c6c8-11e5-9f77-997e28e119fe.png)

![screen shot 2016-01-29 at 7 58 40 pm](https://cloud.githubusercontent.com/assets/1854440/12692667/6b5a3a02-c6c8-11e5-9c1e-35e9bd3b54a7.png)



Fixes #1817

omarjackman commented:
cc @dllh

dllh commented:
Hmm, it didn't work for me:

![chat-formatting](https://cloud.githubusercontent.com/assets/2738252/12753206/1c7c0b9e-c994-11e5-8ae1-95f2eff8670b.png)

Maybe it has to do with the fact that my operator wasn't using an avatar?

I was also curious why the nickname CSS is at play here. It seems like the `display:block` would be all we'd need for this particular fix, but then I know we're also working with some hairy markup and CSS here, so who knows?

omarjackman commented:
@dllh I removed my avatar and retested this on chrome, IE, opera, firefox on windows and chrome, opera, firefox, and safari on osx and couldn't reproduce it. Could you try again and make sure that your not getting a cached version of calypso.

> I was also curious why the nickname CSS is at play here. It seems like the display:block would be all we'd need for this particular fix

I added that little bit to minimize the white space between the notification and the following message.

dllh commented:
I just tested again and it worked. I did a fresh `make run`, so maybe that was the difference. LGTM, let's :ship:.
